client/src/templates/Introduction/components/CertChallenge.js

```diff
@@ -118,10 +145,9 @@ const CertChallenge = ({
       ? verifyCert(certSlug)
       : createFlashMessage(honestyInfoMessage);
   };
-
   return (
     <div className='block'>
-      {(!isCertified || !canViewCert) && (
+      {!isCertified && userLoaded && isSignedIn && (
```

Comment:
Correct me, if I am wrong, but this still needs to hold for if a Camper has not accepted the academic honesty etc.
```suggestion
      {(!isCertified || !canViewCert) && userLoaded && isSignedIn && (
```

Reply:
I hardly could tell the difference between isCertified and canViewCert

Reply:
`isCertified` is when a Camper has claimed a certificate. `canViewCert` is for when a Camper has made the correct settings public to allow a certificate to be viewed.

These two could possibly be combined into one, but at some point we might want slightly different behaviour.